proto/ca_service.proto

```diff
@@ -15,7 +15,14 @@
 syntax = "proto3";
 import "google/rpc/status.proto";
 
-package istio.v1.auth;
+package proto;
```

Comment:
Because the convention is to have the package name and directory name as same. This is what I have seen mixer do. Also, the build is unable to find the package otherwise.